runtime/gc_glue_java/ScavengerRootClearer.cpp

```diff
@@ -106,6 +106,7 @@ MM_ScavengerRootClearer::scavengeReferenceObjects(MM_EnvironmentStandard *env, u
 {
 	Assert_MM_true(env->getGCEnvironment()->_referenceObjectBuffer->isEmpty());
 
+	env->disableHotFieldDepthCopy(); //disable dynamicBreadthFirstScanOrdering depth copying before scavenging reference objects
```

Comment:
move the comment(s) above one line and make them C style
also expand the comment providing a reason (for example to avoid immediate copying of hot children of reference objects)